site/source/docs/api_reference/bind.h.rst

```diff
@@ -691,10 +697,15 @@ Classes
       .. code-block:: cpp
 
          //prototype
-         template<typename Getter, typename Setter>
+         template<typename PropertyType = internal::DeduceArgumentsTag, typename Getter, typename Setter>
          EMSCRIPTEN_ALWAYS_INLINE const class_& property(const char* fieldName, Getter getter, Setter setter) const
 
       This is a function template taking typenames ``Setter`` and ``Getter``: ``template<typename Getter, typename Setter>``
+      which declares a read-write property with the specified ``fieldName`` on the class. ``Getter`` and ``Setter`` may be either a
+      class method, a function, a ``std::function`` or a function object.  When ``Getter`` or ``Setter`` is not pointer-to-member-function,
+      it must accept an instance of the ``ClassType`` as the ``this`` argument. When ``Getter`` or ``Setter`` is a function object, the
+      property type must be specified as a template parameter as it cannot be deduced, e.g.:
+      ``myClass.addProperty<int>("myIntProperty", MyIntGetterFunctor(), MyIntSetterFunctor());``
```

Comment:
Should this `addProperty` in the docs be `property` instead?

Reply:
Ahh, thanks!  Will fix this up.